src/pages/Transactions/Keylist/ActionButton.tsx

```diff
@@ -97,10 +97,7 @@ export const ActionButton = ({
     );
   }
 
-  if (
-    transactionStatus === TransactionStatus.FAILED ||
```

Comment:
We don't allow transactions to be processes if they are `FAILED`, so there shouldn't be a retry button.